test/controllers/check.your.answers.controller.spec.ts

```diff
@@ -650,6 +676,10 @@ describe("GET tests", () => {
     expect(resp.text).toContain(PUBLIC_REGISTER_NAME + " / " + PUBLIC_REGISTER_JURISDICTION + " / " + REGISTRATION_NUMBER);
     expect(resp.text).toContain(SOMEONE_ELSE_REGISTERING);
   });
+
+  test(`renders the ${CHECK_YOUR_ANSWERS_PAGE} page with correct Beneficial Owner natures of control`, async () => {
+    await testPageContainsNaturesOfControl(CHECK_YOUR_ANSWERS_URL);
```

Comment:
it is bizarre to have a test with no expectations
It might be better to use test.each

Reply:
test.each was my first thought but the rest of the CHECK_YOUR_ANSWERS_WITH_PARAMS_URL tests are in a different describe block. I can change to a test.each.

Reply:
updated